src/MSBuildLocator/MSBuildLocator.cs

```diff
@@ -15,6 +15,9 @@
 
 namespace Microsoft.Build.Locator
 {
+    /// <summary>
+    /// MSBuildLocator allows the use of publicly visible MSBuild APIs in custom applications. See our documentation at aka.ms/msbuild/locator.
```

Comment:
This doesn't read much like other static classes, for example https://learn.microsoft.com/en-us/dotnet/api/system.gc?view=net-7.0. I like linking to the conceptual doc, though--is it best to do that here like this, or after the XML ingestion, @ghogen?

```suggestion
    /// Allows enumerating installed MSBuild instances and preparing MSBuild APIs for use.
```